Orange/tests/test_base.py

```diff
@@ -0,0 +1,24 @@
+import unittest
```

Comment:
Please put somewhere at the top of the module:
```py
# Test methods with long descriptive names can omit docstrings
# pylint: disable=missing-docstring
```
And if possible, make sure the [PyLint test](https://travis-ci.org/biolab/orange3/jobs/125246229#L555) passes.